.ddev/commands/web/clone

```diff
@@ -29,6 +60,8 @@ then
   rm -rf "web/modules/contrib/$MODULE"
 fi
 
+# Create the contrib directory if it does not exist.
+mkdir -p web/modules/contrib
```

Comment:
This directory should be created by Drupal on install. Did you run into a case where it wasn't? 

Should we instead drop a `.gitkeep` file in that directory?

Reply:
I went ahead and removed the line. My contrib folder was deleted at some point, but the Drupal install did rebuild the folder.